js_modules/dagster-ui/packages/ui-core/src/overview/OverviewPageHeader.tsx

```diff
@@ -0,0 +1,45 @@
+import {Alert, Box, Heading, PageHeader} from '@dagster-io/ui-components';
+import React, {useContext} from 'react';
+
+import {OverviewTabs} from './OverviewTabs';
+import {CloudOSSContext} from '../app/CloudOSSContext';
+import {useAutoMaterializeSensorFlag} from '../assets/AutoMaterializeSensorFlag';
+import {useStateWithStorage} from '../hooks/useStateWithStorage';
+
+export const OverviewPageHeader = ({
+  tab,
+  queryData,
+  refreshState,
+  ...rest
+}: React.ComponentProps<typeof OverviewTabs> &
+  Omit<React.ComponentProps<typeof PageHeader>, 'title'>) => {
+  const {showGlobalAMPToSensorAMPBanner} = useContext(CloudOSSContext);
+  const [isBannerHidden, setIsBannerHidden] = useStateWithStorage<'1' | undefined>(
+    'banner:showGlobalAMPToSensorAMPBanner',
+    (key) => key,
+  );
+  const isSensorAMP = useAutoMaterializeSensorFlag() === 'has-sensor-amp';
+  return (
+    <PageHeader
+      title={<Heading>Overview</Heading>}
+      tabs={
+        <Box flex={{direction: 'column', gap: 8}}>
+          <OverviewTabs tab={tab} queryData={queryData} refreshState={refreshState} />
+          {isSensorAMP && showGlobalAMPToSensorAMPBanner && isBannerHidden !== '1' ? (
+            <Alert
+              title={
+                "The 'Auto-materialize' tab has been removed due to opting into automation sensors."
```

Comment:
They're called "Auto-materialize sensors now" and ideally we can use this same banner in a future world where it is opt-out instead of opt-in

How about something like "Looking for the `Auto-materialize` tab?"
"You can find your auto-materialize sensors on the `Sensors` tab."

Reply:
+1 to Dan's suggestions

Also, no single quotes needed around the tab names

Reply:
Updated!